node/pkg/telemetry/prom_remote_write/scrape.go

```diff
@@ -0,0 +1,97 @@
+package promremotew
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/golang/snappy"
+)
+
+func scrapeLocalMetrics() ([]byte, error) {
+	// The idea is to grab all the metrics from localhost:6060/metrics,
+	// and then send them to Grafana.
+	const metricsPort = 6060
```

Comment:
The `6060` comes from the `statusAddr` node config parameter, right? Should you extract the port number from there and pass it in?

Reply:
Changed.